environment-pipelines/locals.tf

```diff
@@ -5,6 +5,8 @@ locals {
     managed-by          = "DBT Platform - Terraform"
   }
 
+  # Todo: This appears unused to tflint. Can it be safely deleted?
+  # tflint-ignore: terraform_unused_declarations
```

Comment:
@antroy-madetech / @JohnStainsby I can't find any uses of this. Are either of you able to shed any light on whether this is actually needed?

Reply:
Easiest way to heck would be remove it and see if the linting pre-commit check passes.

Reply:
@antroy-madetech linting pre-commit removes the following code as the variable is unused:
`stage_config = yamldecode(file("${path.module}/stage_config.yml"))`

tests in environment-pipelines still pass once the variiable is removed

Reply:
I guess we can remove `stage_config.yml` too then